linode/manager
# fix: [M3-7864] - Make the ACL (Object storage) select field carat (^) symbol consist with other select fields in the CM.

| Before  | After   |
| ------- | ------- |
|![image](https://github.com/linode/manager/assets/119517080/df2382d0-20e1-4698-ac49-f016ba450df0) | ![image](https://github.com/linode/manager/assets/119517080/444d91e6-34a3-448c-b7aa-fd3ff193a6e0) |

## How to test 🧪
### Verification steps
(How to verify changes)
- Navigate to Object storage bucket details Access tab and verify updating ACL.
- Navigate to Object details drawer and verify updating ACL.
- Verify `AccessSelect` references and ensure there is no regression. 

## As an Author I have considered 🤔

*Check all that apply*
